metrics/report/report_dockerfile/scaling.R

```diff
@@ -229,49 +221,40 @@ mem_line_plot <- ggplot(data=fndata, aes(as.numeric(as.character(pod)),
 		as.numeric(as.character(mem_free)),
 		colour=testname, group=interaction(testname, node))) +
 	geom_line(alpha=0.2) +
-	xlab("Pods") +
+	geom_point(aes(shape=node), alpha=0.3, size=0.5) +
+	xlab("pods") +
 	ylab("System Avail (Gb)") +
 	scale_y_continuous(labels=comma) +
 	ggtitle("System Memory free") +
 	#ylim(0, NA) + # For big machines, better to not 0-index
 	theme(axis.text.x=element_text(angle=90))
 
-# If we only have relatively few samples, add points to the plot. Otherwise, skip as
-# the plot becomes far too noisy
-if ( skip_points == 0 ) {
-	mem_line_plot = mem_line_plot + geom_point(aes(shape=node), alpha=0.3)
-}
-
 cpu_stats_plot = suppressWarnings(ggtexttable(data.frame(cstats), theme=ttheme(base_size=10), rows=NULL))
 
 # plot how samples varied over 'time'
 cpu_line_plot <- ggplot(data=fndata, aes(as.numeric(as.character(pod)),
 		as.numeric(as.character(cpu_idle)),
 		colour=testname, group=interaction(testname, node))) +
 	geom_line(alpha=0.2) +
-	xlab("Pods") +
+	geom_point(aes(shape=node), alpha=0.3, size=0.5) +
+	xlab("pods") +
 	ylab("System CPU Idle (%)") +
 	ggtitle("System CPU usage") +
 	#ylim(0, NA) + # For big machines, better to not 0-index
 	theme(axis.text.x=element_text(angle=90))
 
-if ( skip_points == 0 ) {
-	cpu_line_plot = cpu_line_plot + geom_point(aes(shape=node), alpha=0.3)
-}
+cpu_line_plot = cpu_line_plot + 
```

Comment:
this line left over from a search/delete maybe? Line no longer needed (and is not valid syntax) I think.